libs.jstestdriver/external/jstestdriver-1.3.5-license.txt

```diff
@@ -0,0 +1,20 @@
+Name: JS Test Driver
+Version: 1.3.5
+License: Apache-2.0-REF
```

Comment:
Given, that the referenced license is plain Apache-2.0, I would use that. From my POV it does not matter, that google decided to just reference it, but the intention is clear. The Copyright info is also not relevant here, as the Apache license only requires parts of NOTICE to be repeated.

The binary also bundles several third party jars - it might be worth adding a "Comment: ..." header, that explains, that the file is only downloaded to be part of the classpath when the extension file is compiled (i.e. we don't plan to redistribute it).

Reply:
@matthiasblaesing sounds reasonable not to clutter the license list further, thanks.